### PR TITLE
docs: add token being broken if team member leaves org

### PR DESCRIPTION
**Pull request by pedpess**
Add a section that helps with troubleshooting in the event of a team member leaving the company and having its e-mail deactivated. That may cause AppCenter not recognizing the token anymore if the member was the one that created it.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@pedpess : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/distribution/uploading.md [hunk 1]
```diff
@@ -126,3 +126,11 @@ You can find links to specific releases to public destinations on the releases t
 [app-center-home]: https://appcenter.ms/apps
 [apple-register-single-device]: https://help.apple.com/developer-account/#/dev40df0d9fa
 [apple-register-multiple-devices]: https://help.apple.com/developer-account/#/devebd34abb1
+
+## Troubleshooting
+
+### I'm getting "404: Not Found" when I try to upload my binaries to AppCenter distribution
+
+This error can happen if a member of your company got its corporate e-mail deactivated. Not having the member inside of the AppCenter's org, but also if the member was the one who created the API tokens to publish to AppCenter, it might end-up with the token getting rejected.
```
Comment (King-of-Spades) [inline docs/distribution/uploading.md:134]: The description here is very confusing. I (think)? you're trying to warn readers that if someone's account is deactivated in App Center, then their User API tokens would no longer be accessible? 

That's a bit different from your word choice "got its corporate e-mail deactivated." Normally, if an email account is deactivated outside of App Center, our system wouldn't know about the deletion and thus deactivate/delete relevant user assets.

**Review by jzeferino — APPROVED** [review]

File: docs/distribution/uploading.md (hunk 1 above)
Comment (King-of-Spades) [reply docs/distribution/uploading.md:134]: P.S. I originally wrote this feedback much closer to when you originally created this PR, but I must've clicked away or the wrong thing since I just stumbled back here and saw it was a "saved draft." Sorry for the late feedback.

Comment (pedpess) [reply docs/distribution/uploading.md:134]: hey, @King-of-Spades no problem :D ... Yes, you are correct in this part "got its corporate e-mail deactivated."... We had a co-worker that made the tokens using his Gmail account. Once he left the company the e-mail was deactivated and pushing binaries to AppCenter stopped working.

Would you have a recommendation for a sentence that would explain this situation better? I can change it this week and send the corrections to you if you have something.

Comment (King-of-Spades) [reply docs/distribution/uploading.md:134]: I'd recommend the following changes:
- The header should be talking about the email deactivation aspect, and the 404 error context should be in the body, because 404 errors have so many causes for things not being found. 
- Instead of "a member/corporate e-mail deactivated" it should more explicitly state something like "the (user or account) who created the tokens" or "configured the distribution" etc. 

Basically just making sure we have a really clear connection between cause & effect.

Comment (King-of-Spades) [reply docs/distribution/uploading.md:134]: I can also polish a bit directly once we're on the same page. (Since I don't precisely have a reproduction, it's tricky for me to accurately rewrite it at present.)

**Comment by pedpess** [comment]
Hi again :)… Thanks for the recommendations and sorry for the delay. I will try to update it this weekend.
